test/results.el

```diff
@@ -1739,6 +1863,13 @@ Wednesday   5 July 2017
 " "25b5f6da5442f760584293ee1a07e0e8" "Day-agenda (W27):
 Wednesday   5 July 2017
 
+ Face: default
```

Comment:
This one is also caused by a change in Org.

Reply:
I have now fixed this one too.